operations/security_and_authentication/keycloak_ipv6_support.md

```diff
@@ -0,0 +1,90 @@
+# Keycloak IPv6 Support
+
+## IPv6 support
+
+The `cray-keycloak` service can be configured to access an external LDAP server using IPv6. CSM does not deploy Kubernetes in dual stack mode so this is achieved
+by using a network attachment definition to allow the `cray-keycloak` pods direct access to an IPv6 network.
+
+The following `cray-keycloak` Helm chart values must be set to enable IPv6 support.
+
+| Property          | Default value | Description                                                                                                 |
+| ----------------- | ------------- | ----------------------------------------------------------------------------------------------------------- |
+| `ipv6.enabled`    | false         | Enable/Disable IPv6 support                                                                                 |
+| `ipv6.gateway`    | None          | The default gateway to use for IPv6 traffic. Must be set if `ipv6.enabled=true`                             |
+| `ipv6.subnet`     | None          | The IPv6 subnet to use in CIDR form. Must be set if `ipv6.enabled=true`                                     |
+| `ipv6.rangeStart` | None          | Start address of an IPv6 address pool to be used for `cray-keycloak`. Must be set if `ipv6.enabled=true`    |
+| `ipv6.rangeEnd`   | None          | End address of an IPv6 address pool to be used for `cray-keycloak`. Must be set if `ipv6.enabled=true`      |
+
+The values used for `ipv6.gateway` and `ipv6.subnet` should match those used for the Customer Management Network (CMN). The `ipv6.rangeStart` and `ipv6.rangeEnd` values
+should describe an unused range within the subnet declared in `ipv6.subnet`. The number of IP addresses in this range should equal the desired number of `cray-keycloak`
+replicas (the default is 3). If `cray-dns-unbound` has also been configured to use IPv6, then this range *must not* overlap with the range used there.
+
+## Prerequisites
+
+- CSM must have been configured to support IPv6 on the CMN. See the [IPv6 Configuration Guide](../network/customer_accessible_networks/ipv6_configuration_guide.md) for more information.
+
+## Procedure
+
+**NOTE:** This procedure assumes that CSM has already been installed and a running system is being modified. If the system
```

Comment:
Should we not also put something in Prepare Site Init that points back to this page? Or includes this (optional) step?

Reply:
And is there anything we should put in the upgrade docs for this? Or in that scenario, is the assumption that they'll either do this before or after the upgrade?